sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -151,8 +141,11 @@ void PeriodicExportingMetricReader::DoBackgroundWork()
       worker_thread_instrumentation_->AfterWait();
     }
 #endif /* ENABLE_THREAD_INSTRUMENTATION_PREVIEW */
-
-  } while (IsShutdown() != true);
+    if (IsShutdown())
```

Comment:
With the `return` changed back to `break`, is it the same as the original `do...while` loop? If so, just restore it to the `do...while` loop which seems more concise?